runner/android_junit_runner/java/androidx/test/internal/runner/TestRequestBuilder.java

```diff
@@ -78,7 +80,8 @@ public class TestRequestBuilder {
           .intersect(new SdkSuppressFilter())
           .intersect(new RequiresDeviceFilter())
```

Comment:
@brettchabot would it make sense to reimplement these as CustomFilter too?

Also is there a way to leave comments on these PRs so that the author sees them?

Reply:
never mind https://github.com/android/android-test/pull/1366/files,
but the second part of the question still stands